iceoryx_posh/include/iceoryx_posh/mepoo/chunk_header.hpp

```diff
@@ -161,7 +161,7 @@ struct ChunkHeader
     uint8_t m_reserved{0};
     // currently just a placeholder
     uint16_t m_userHeaderId{NO_USER_HEADER};
-    UniquePortId m_originId{popo::InvalidId};
+    popo::UniquePortId m_originId{popo::InvalidPortId};
```

Comment:
Shouldn't a chunkheader have always a valid port id? I think it may makes more sense to get rid of the default value here, remove the default ctor from the unique port id (if there is one) and force to set it here correctly.

If the chunkheader can have an invalid port id we should turn this into a `optional<popo::UniquePortId>` instead.

Reply:
It is also used in the `TypedMemPool`, which doesn't have a port ID. Therefore an `optional` would be the best solution